Formula/y/yt-dlp.rb

```diff
@@ -3,9 +3,9 @@ class YtDlp < Formula
 
   desc "Feature-rich command-line audio/video downloader"
   homepage "https://github.com/yt-dlp/yt-dlp"
-  url "https://files.pythonhosted.org/packages/70/1d/f9617f151c124a15981cf79c47b6a53ac875942211b40010f0381296c51d/yt_dlp-2024.3.10.tar.gz"
-  version "2024.03.10"
-  sha256 "6e74cb14a69dbeb872c8ef4e0b8bbed2ee846ec633513cf3124a74c1faedc07b"
+  url "https://files.pythonhosted.org/packages/52/85/af44c30f0cc5d94ebbce365f8ee40eea6e9d7a1d26d50f16e3766ca8410c/yt_dlp-2024.4.9.tar.gz"
+  version "2024.04.09"
```

Comment:
maybe remove this?
```suggestion
```

Reply:
I think it's required because the GitHub tag and the version on pypi differ.

Reply:
where is the reference about such requirement?

---

I just did some local testing, it works fine removing it.

Reply:
we also dont assert the version, even though after removing it, the version output is still `2024.04.09`

Reply:
the version field is actually introduced by autobump, a bit weird, #165689

Reply:
The filename in that PR doesn't have the leading 0 for the month, which is probably why.